contracts/FullCheckpoint.sol

```diff
@@ -64,26 +64,21 @@ contract FullCheckpoint {
         (bytes32 ph, int256 n) = HeaderReader.getParentHashAndNumber(
             genesisHeader
         );
-        (bytes32 ph1, int256 n1, uint64 rn) = HeaderReader.getBlock1Params(
-            block1Header
-        );
-        require(n == 0 && n1 == 1, "Invalid Init Block");
+
+        HeaderReader.ValidationParams memory block1 = HeaderReader
+            .getValidationParams(block1Header);
+        require(n == 0 && block1.number == 1, "Invalid Init Block");
         headerTree[genesisHeaderHash] = Header({
+            receiptHash: bytes32(0),
```

Comment:
is it really `block0.receiptHash = bytes32(0)`?

Reply:
fix